### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by Creol3Kid**
just a pull request

### CODE REVIEW

File: practice_project/main.py [hunk 1]
```diff
@@ -1,26 +1,16 @@
...
+
+def is_prime(number):
+    if number <= 1:
         return False
-    for i in range(2, int(math.sqrt(n)) + 1):
-        if n % i == 0:
+    for i in range(2, int(math.sqrt(number)) + 1):
+        if number % i == 0:
             return False
     return True
```
Comment (leonstaubach) [inline practice_project/main.py:10]: you should separate any source files from test files, it's industry standard to not mix the two concepts up in the same files

File: practice_project/main.py [hunk 2]
```diff
@@ -33,6 +23,18 @@ def test_is_prime(self):
...
+		self.assertTrue(is_prime(3))
+		self.assertTrue(is_prime(5))
+		self.assertTrue(is_prime(7))
+		self.assertTrue(is_prime(11))
+		self.assertTrue(is_prime(13))
+		self.assertTrue(is_prime(17))
+		self.assertTrue(is_prime(19))
+		self.assertTrue(is_prime(23))
+		self.assertTrue(is_prime(29))
+		self.assertTrue(is_prime(31))
```
Comment (leonstaubach) [inline practice_project/main.py:37]: instead of one big test case, i suggest breaking it down into multiple separate test cases. also look for edge cases (input smaller 0, very large, unexpected format)

**Review by leonstaubach — COMMENTED** [review]
functionally looks all good! some refactor necessary for readability and QA
